services/FacebookPayments.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "FacebookPayments",
+  "documents": {
+    "tos": {
+      "location": "https://www.facebook.com/payments_terms",
+      "contentSelector": ".prl",
+      "filters": [
+        "removeHelpButtons",
+        "cleanUrls"
+      ]
+    },
+    "devPolicy": {
+      "location": "https://developers.facebook.com/policy/credits",
+      "contentSelector": "._4bl9",
+      "filters": [
+        "cleanUrls"
```

Comment:
There is a weird numbering in the Payments Developer Policy, as well as improperly parsed titles.

![Capture d’écran 2020-08-05 à 16 36 54](https://user-images.githubusercontent.com/222463/89425994-f151d080-d739-11ea-8bb1-1cedf7710680.png)

Reply:
Created a numberListProperly to adress this issue.
Strangely the HTML have badly formatted <ol> <li> tags, which interferes with proper numbering anyway :disappointed: 
What do you mean by improperly parsed titles ?

Reply:
> 1. What do you mean by improperly parsed titles ?

Line 9: h3 in a list.

Not blocking.

Reply:
It should be solved if I understood well